Fix build failure with OS X 10.11 SDK earlier than 10.11.4 (Xcode 7.2.1 and earlier).
--- Frameworks/DocumentWindow/src/SelectGrammarViewController.mm.orig
+++ Frameworks/DocumentWindow/src/SelectGrammarViewController.mm
@@ -21,7 +21,7 @@
 @property (nonatomic) OakDocumentView*     documentView;
 @property (nonatomic) BundleGrammar*       grammar;
 
-@property (nonatomic) void(^callback)(SelectGrammarResponse, BundleGrammar*);
+@property (nonatomic, strong) void(^callback)(SelectGrammarResponse, BundleGrammar*);
 @end
 
 static NSButton* OakSmallButton (NSString* title, SEL action, id target, NSInteger tag)
--- Frameworks/OakCommand/src/OakCommand.h.orig
+++ Frameworks/OakCommand/src/OakCommand.h
@@ -13,8 +13,8 @@ NS_ENUM(NSInteger) {
 PUBLIC @interface OakCommand : NSObject
 @property (nonatomic, weak) NSResponder* firstResponder;
 @property (nonatomic, readonly) NSUUID* identifier;
-@property (nonatomic) void(^modalEventLoopRunner)(OakCommand*, BOOL* didTerminate);
-@property (nonatomic) void(^terminationHandler)(OakCommand*, BOOL normalExit);
+@property (nonatomic, strong) void(^modalEventLoopRunner)(OakCommand*, BOOL* didTerminate);
+@property (nonatomic, strong) void(^terminationHandler)(OakCommand*, BOOL normalExit);
 @property (nonatomic) BOOL updateHTMLViewAtomically;
 @property (nonatomic, readonly) OakHTMLOutputView* htmlOutputView;
 - (instancetype)initWithBundleCommand:(bundle_command_t const&)aCommand;
